packs/teamcity/actions/build_job.py

```diff
@@ -0,0 +1,12 @@
+from lib.action import TeamCityAction
+
+
+class BuildJobAction(TeamCityAction):
+
+    def run(self, build_config_id, branch_name=None):
+        request = '''<build branchName="%s">
```

Comment:
For safety and robustness reasons (to avoid XML injections, etc.) please use ElementTree for building XML documents - https://docs.python.org/2/library/xml.etree.elementtree.html